src/ripple/protocol/jss.h

```diff
@@ -160,6 +160,7 @@ JSS(alternatives);           // out: PathRequest, RipplePathFind
 JSS(amendment_blocked);      // out: NetworkOPs
 JSS(amendments);             // in: AccountObjects, out: NetworkOPs
 JSS(amm);                    // out: amm_info
+JSS(amm_account);            // in: amm_info
```

Comment:
I thought that was pretty clear. But I'm not good with names. @mDuo13 what do you think about the name change?

Reply:
Both seem fine to me

Reply:
Yes, this is correct.

Reply:
Expanding on this for future readers and tech writers / documenters:

- `amm_account` is AMM account ID. It allows users to fetch AMM Object with AMM account ID instead of using the asset pair.
- Either `amm_account` or `asset` and `asset2` have to be included.
- `account` is a liquidity provider (LP) account. If `account` is included, then `lp_token` is set to that LP's token balance.
  - `account` is an `r...` address and is kind of like the "perspective" account.
  - It is like the `peer` in `account_lines`, or the `taker` in `book_offers`.